app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java

```diff
@@ -51,9 +61,17 @@
     private TextView txtvPodcastTitle;
     private TextView txtvEpisodeTitle;
     private ImageView imgvCover;
+    private ImageButton openDescription;
+    private LinearLayout openDescriptionLayout;
+    private FrameLayout counterweight;
+    private FrameLayout spacer;
+    private ImageButton butPrevChapter;
+    private ImageButton butNextChapter;
+    private LinearLayout episodeDetails;
+    private LinearLayout chapterControl;
     private PlaybackController controller;
     private Disposable disposable;
-    private int displayedChapterIndex = -2;
+    private int displayedChapterIndex = -1;
```

Comment:
The idea of the -2 was that it force-refreshes the data. Otherwise, we sometimes do not update the chapter image when switching to a new file while the player is closed (if I remember correctly). Not 100% sure if the same problem happens here, too, but it would be worth testing similar scenarios

Reply:
hmm i dont know, can you test? i didnt change anything of how chapters a handled.

Reply:
i think this is from the PR of @jonasburian

Reply:
I think it was -2 because of the condition [here](https://github.com/AntennaPod/AntennaPod/pull/5075/files#diff-14d98525c5df8d24fa975f628614e462438e8ccb86a08dab5f5bafabb2ef09cfL150) in line 150. If there are no chapters, `ChapterUtils.getCurrentChapterIndex()` would both be -1, but since this condition is now away, this should no longer be a problem I think. The image will be refreshed in `onCreateView()` -> `refreshChapterData()` anyway.